codegen/main.go

```diff
@@ -40,7 +38,7 @@ namespace Nakama
     using System.Text;
     using System.Threading;
     using System.Threading.Tasks;
-    using TinyJson;
+    using Nakama.TinyJson;
```

Comment:
We should bundle a separate copy of TinyJson with Satori to remove the dependence between Satori and Nakama namespaces.